label_studio/organizations/models.py

```diff
@@ -141,6 +148,11 @@ def per_project_invited_users(self):
         per_project_invited_users = User.objects.filter(pk__in=invited_ids)
         return per_project_invited_users
 
+    def should_verify_ssl_certs(self) -> bool:
```

Comment:
we have similar functionality in licenses: 
https://github.com/HumanSignal/label-studio-enterprise/blob/develop/label_studio_enterprise/billing/models.py#L45

better to use the same mechanics

Reply:
agreed, I'll remove the Organization field and migration, great call here :+1: